main.go

```diff
@@ -954,8 +956,9 @@ func transformLink(args []string) ([]string, error) {
 			pkgPath = buildInfo.firstImport
 		}
 		if id := buildInfo.imports[pkgPath].buildID; id != "" {
+			garbledPkg := garbledImports[pkg]
```

Comment:
Might be handling buildIDs wrong in `obfuscateImports()`, as using `pkgPath` here didn't work @mvdan

Reply:
it depends on whether you want the top-level main package to have path `main` or its "true" path like `mvdan.cc/garble`. whatever works is fine, really, because both can be valid answers depending on who asks.